cmd/minikube/cmd/start.go

```diff
@@ -471,9 +471,9 @@ func validateUser() {
 	d := viper.GetString(vmDriver)
 	// Check if minikube needs to run with sudo or not.
 	if err == nil {
-		if d == constants.DriverNone && u.Name != "root" {
+		if d == constants.DriverNone && u.Username != "root" {
 			exit.UsageT(`Please run with sudo. the vm-driver "{{.driver_name}}" requires sudo.`, out.V{"driver_name": constants.DriverNone})
-		} else if u.Name == "root" && !(d == constants.DriverHyperv || d == constants.DriverNone) {
+		} else if u.Username == "root" && !(d == constants.DriverHyperv || d == constants.DriverNone) {
```

Comment:
Thank you! This should really be checking for `u.Uid == 0`, as systems can have multiple root-access accounts with a uid of 0.

The root user can also be renamed.